### PR TITLE
[SMALLFIX] Add mount points to master info REST endpoint.

### PR DESCRIPTION
**Pull request by uronce-cc**

### CODE REVIEW

File: core/server/src/main/java/alluxio/master/AlluxioMasterRestServiceHandler.java [hunk 1]
```diff
@@ -134,10 +139,10 @@ public AlluxioMasterInfo call() throws Exception {
   }
 
   /**
-   * @summary get the configuration map, the keys are ordered alphabetically.
    * @return the response object
-   * @deprecated since version 1.4 and will be removed in version 2.0
+   * @summary get the configuration map, the keys are ordered alphabetically.
```
Comment (jsimsa) [inline core/server/src/main/java/alluxio/master/AlluxioMasterRestServiceHandler.java]: why is summary after return? should javadoc for other REST API handler files be changed accordingly

Comment (uronce-cc) [inline core/server/src/main/java/alluxio/master/AlluxioMasterRestServiceHandler.java]: auto formatted by IDE, reordered them.

**Comment by jsimsa** [comment]
please fix the failing test

**Comment by jsimsa** [comment]
I don't think we should be using the actual UFS data

**Comment by apc999** [comment]
@uronce-cc is this PR ready to review? I see test is still failing

**Comment by uronce-cc** [comment]
jenkins, test this please

**Comment by uronce-cc** [comment]
@apc999 Tests are fixed.

File: tests/src/test/java/alluxio/master/AlluxioMasterRestApiTest.java [hunk 2]
```diff
@@ -106,6 +111,19 @@ public void getMetrics() throws Exception {
   }
 
   @Test
+  public void getMountPoints() throws Exception {
+    Map<String, MountInfo> mountTable = mFileSystemMaster.getMountTable();
+    Map<String, MountPointInfo> mountPoints = getInfo(NO_PARAMS).getMountPoints();
+    Assert.assertEquals(mountTable.size(), mountPoints.size());
+    for (Map.Entry<String, MountInfo> mountPoint : mountTable.entrySet()) {
+      Assert.assertTrue(mountPoints.containsKey(mountPoint.getKey()));
+      String wantUri = mountPoints.get(mountPoint.getKey()).getUfsUri();
```
Comment (apc999) [inline tests/src/test/java/alluxio/master/AlluxioMasterRestApiTest.java]: NIT: `expectedUri`

File: tests/src/test/java/alluxio/master/AlluxioMasterRestApiTest.java [hunk 3]
```diff
@@ -106,6 +111,19 @@ public void getMetrics() throws Exception {
...
+      String gotUri = mountPoint.getValue().getUfsUri().toString();
```
Comment (apc999) [inline tests/src/test/java/alluxio/master/AlluxioMasterRestApiTest.java]: NIT: `returnedUri`

File: tests/src/test/java/alluxio/master/AlluxioMasterRestApiTest.java (hunk 2 above)
Comment (uronce-cc) [inline tests/src/test/java/alluxio/master/AlluxioMasterRestApiTest.java]: done

**Pull request merged**